cypress-tests/cypress/support/commands.js

```diff
@@ -763,30 +781,31 @@ Cypress.Commands.add(
 Cypress.Commands.add(
   "createPaymentIntentTest",
   (
-    request,
+    createPaymentBody,
     req_data,
     res_data,
     authentication_type,
     capture_method,
     globalState
   ) => {
     if (
-      !request ||
-      typeof request !== "object" ||
-      !req_data.currency ||
-      !authentication_type
```

Comment:
Was this verified with @Sakilmostak?

Reply:
Yes verified and no use of authentication_type so i have removed it